bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -225,6 +225,13 @@ void recv_callback(void *request, ucs_status_t status, ucp_tag_recv_info_t *info
     jucx_request_callback(request, status);
 }
 
+void stream_recv_callback(void *request, ucs_status_t status, size_t length)
+{
+    struct jucx_context *ctx = (struct jucx_context *)request;
+    ctx->length = length;
+    jucx_request_callback(request, status);
```

Comment:
should we set recv_size_field?

Reply:
It sets in `jucx_request_callback`: https://github.com/openucx/ucx/blob/master/bindings/java/src/main/native/jucx_common_def.cc#L214-L217